README.md

```diff
@@ -33,7 +33,7 @@ repository after the new release of your package is merged into the registry.
 ### Requirements for new packages
 
 New packages that meet the following requirements will be eligible for automatic merging (see below).
-Packages that do not meet these requirements will be subject to a longer period of community review.
+New packages that do not meet these requirements will be subject to a longer 7-day waiting period (compared to a 3-day waiting period for packages that meet these requirements). 
```

Comment:
See my edits to my response to your question.

Reply:
I'll add the "required an approving review" by a commiter.

What do you think about 10 days instead of 7 days?

Reply:
10 days is better.